src/commands/android/utils/sdk.ts

```diff
@@ -224,6 +224,28 @@ export const execBinaryAsync = (
   });
 };
 
+export const spawnCommandSync = (binaryLocation: string, binaryName: string, platform: Platform, args: string[]) => {
+  let cmd: string;
+  if (binaryLocation === 'PATH') {
+    const binaryFullName = getBinaryNameForOS(platform, binaryName);
+    cmd = `${binaryFullName}`;
+  } else {
+    const binaryFullName = path.basename(binaryLocation);
+    const binaryDirPath = path.dirname(binaryLocation);
+    cmd = path.join(binaryDirPath, binaryFullName);
```

Comment:
Why not just do `cmd = binaryLocation` here?

Reply:
That's right. I don't know why I wrote that in the first place.